tests/integration/metrics/multicolumn_values/test_sum_equal.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+
+from great_expectations.datasource.fluent.interfaces import Batch
+from great_expectations.metrics.multi_column_values.sum import (
+    MultiColumnSumEqualUnexpectedCount,
+    MultiColumnSumEqualUnexpectedCountResult,
+)
+from tests.integration.conftest import parameterize_batch_for_data_sources
+from tests.integration.metrics.conftest import (
+    PANDAS_DATA_SOURCES,
+    SPARK_DATA_SOURCES,
+    SQL_DATA_SOURCES,
+)
+
+COL_A = "A"
+COL_B = "B"
+COL_A_WITH_NULLS = "A_WITH_NULLS"
+COL_B_WITH_NULLS = "B_WITH_NULLS"
+DATA_FRAME = pd.DataFrame(
+    {
+        COL_A: [1, 2, 3, 4],
+        COL_B: [4, 3, 2, 1],
+    },
+)
+SUCCESS_SUM_TOTAL = 5
+DATA_FRAME_WITH_NULLS = pd.DataFrame(
```

Comment:
This isn't used. Maybe there is a test commit that didn't get pushed up for testing failures? Also it would be nice to test that we catch it when 2 rows don't equal to 5 but neither is `None`.